app/src/main/java/edu/uc/reedws/musiclink/MainActivity.kt

```diff
@@ -94,8 +94,8 @@ class MainActivity : AppCompatActivity() {
                 "You added " + newPlaylistName.text.toString(),
                 Toast.LENGTH_SHORT
             ).show()
-            val newPlaylistName = newPlaylistName.text
-            viewModel.createPlaylist(newPlaylistName.toString())
+            val playlistName = newPlaylistName.text
+            viewModel.createPlaylist(playlistName.toString())
```

Comment:
As mentioned in my pull request, this solves the build warning for Name Shadowing. I included a link in my pull request to a Stack Overflow forum that offered a little more explanation for this kind of change!

Reply:
Do you need to store it in a variable (playlistName) at all?  You can probably consolidate this all to one line, which makes the intermediate variable unnecessary.